MsCorePkg/MacAddressEmulationDxe/Test/MacAddressEmulationDxeHostTestMain.c

```diff
@@ -0,0 +1,166 @@
+/** @file
+  This module tests MacAddressEmulation behavior for
+  MacAddressEmulationDxe driver.
+
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include "MacAddressEmulationDxe.h"
+
+#include "SnpSupportsMacEmuCheckTest.h"
+#include "FindMatchingSnpTest.h"
+#include "SimpleNetworkProtocolNotifyTest.h"
+#include "EntryPointTest.h"
+
+#define UNIT_TEST_NAME     "Mac Address Emulation Dxe Host Test"
+#define UNIT_TEST_VERSION  "0.1"
+
+EFI_RUNTIME_SERVICES  mMockRuntime;
```

Comment:
Is this maybe a holdover from a copy/paste to start coding?  I don't see anywhere that links to this and it doesn't appear you are pulling in the UnitTest MockUefiRuntimeServicesTableLib.

Reply:
Yes a holdover, will double check if it compiles then remove if necessary.